src/Microsoft.AspNetCore.Authorization/AuthorizationOptions.cs

```diff
@@ -47,6 +60,11 @@ public void AddPolicy(string name, Action<AuthorizationPolicyBuilder> configureP
             PolicyMap[name] = policyBuilder.Build();
         }
 
+        /// <summary>
+        /// Return the policy for the given name, or null.
+        /// </summary>
+        /// <param name="name"></param>
```

Comment:
You still need to expand.

/// <summary>
/// Returns the policy for the specified name, or null if a policy with the name does not exist.
/// </summary>
/// <param name="name">The name of the policy to return.</param>
/// <returns>The policy for the specified name, or null if a policy with the name does not exist.</returns>